$OpenBSD: patch-lib_menu_in,v 1.1 2002/01/20 19:54:13 naddy Exp $
# Add some additional entries to start menu
# Only installed applications will be shown in the menu.
#
# If I can get permission to add more stuff here, then I can list
# almost every X app that is available for OpenBSD.
# I can create the appropriate icons too (made some of them already),
# but that will greatly increase number of files in this port.
# I will try to talk to IceWM developers about adding more applications
# from OpenBSD ports tree to the default menu.
#
--- lib/menu.in.orig	Tue Oct  9 09:54:03 2001
+++ lib/menu.in	Sun Jan 13 20:02:35 2002
@@ -5,6 +5,8 @@
 # (re)install icewm.
 #
 prog xterm xterm xterm
+prog Eterm Eterm Eterm
+prog aterm aterm aterm
 prog rxvt xterm rxvt -bg black -cr green -fg white -C -fn 9x15 -sl 500
 prog fte fte fte
 prog Netscape netscape netscape
@@ -12,6 +14,8 @@ prog Gimp gimp gimp
 separator
 menu Applications folder {
     menu Editors folder {
+        prog vi vi xterm -e vi
+        prog mg mg xterm -e mg
         prog fte fte fte
 	prog vim vim gvim 
         prog xemacs xemacs xemacs
@@ -21,15 +25,26 @@ menu Applications folder {
     }
     menu "WWW Browsers" folder {
         prog Netscape netscape netscape
+        prog Konqueror konqueror konqueror
+        prog Amaya amaya amaya
         prog Arena arena arena
         prog Lynx lynx xterm -e lynx
     }
+    menu "Mail and News" folder {
+        prog Sylpheed sylpheed sylpheed
+        prog TkRat tkrat tkrat
+    }
     menu Graphics folder {
         prog Gimp gimp gimp
         prog XV xv xv
         prog XPaint xpaint xpaint
         prog XFig xfig xfig
     }
+    menu Multimedia folder {
+        prog XPlayCD xplaycd xplaycd
+        prog XMixer xmixer xmixer
+        prog XMMS xmms xmms
+    }
     menu Devel folder {
         prog ddd ddd ddd
     }
@@ -46,15 +61,12 @@ menu Games folder {
     prog XDemineur xdemineur xdemineur
 }
 menu System folder {
-    prog "Control Panel" redhat control-panel
+    prog top top xterm -e top
     prog xload xload xload
     prog xosview xosview xosview
 }
 menu Utilities folder {
-    menu Multimedia folder {
-        prog XPlayCD xplaycd xplaycd
-        prog XMixer xmixer xmixer
-    }
+    prog Xman xman xman
     prog "Font Selector" xfontsel xfontsel
     prog Clock xclock xclock
     prog Magnify xmag xmag
@@ -73,4 +85,7 @@ menu "Window Managers" folder {
     restart blackbox - blackbox
     restart fvwm2 - fvwm2
     restart fvwm - fvwm
+    restart wm2 - wm2
+    restart KDE - startkde
+    restart Golem - golem
 }
